### ISSUE TITLE
Linux: UMS does not start with the included Java installation

### ISSUE THREAD

**Issue by M-itch** [UniversalMediaServer/UniversalMediaServer#1835]
I made a few changes to properly detect and run with the included JRE.

Installed version: 9.0.0
OS: Arch Linux

Fix "cannot execute: Permission denied" error
```
chmod +x linux/jre-x64/bin/java
```

UMS.sh
```# Use our JVM if it exists
if [ "$(uname -m | grep '64')" != "" ]; then
    if [ -f linux/jre-x64/bin/java ]; then 
        JAVA="linux/jre-x64/bin/java"
    fi
else
    if [ -f linux/jre-x86/bin/java ]; then 
        JAVA="linux/jre-x86/bin/java"
    fi
fi
```

**Comment by SubJunk** [comment]
@M-itch thanks a lot, I have applied the change to UMS.sh now. I wonder how we can automate that `chmod` part.

**Comment by M-itch** [comment]
`chmod` won't change the permission when I run it without sudo (depending on the install location).
So we can't add it the UMS.sh after detecting the included JRE.
The packaged release does have the execute permission for the included linux binaries.

I just found out that we can check if the included java binary is executable.
At least then UMS will start with the installed Java without any errors.
```
if [ -x linux/jre-x86/bin/java ]; then
```

**Comment by SubJunk** [comment]
@M-itch I made a PR to hopefully fix this, are you able to test it? If you need me to build and upload it let me know https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1837

**Comment by M-itch** [comment]
Yes, please upload a build for me. I tried to create my own build, but I get a maven plugin (API incompatibility) error while installing the external libraries .

**Comment by SubJunk** [comment]
Thanks for confirming. We will release this soon
